requirements.in

```diff
@@ -5,3 +5,5 @@ djangorestframework
 pandas
 psycopg2-binary
 python-dotenv
+recordlinkage
+xlrd
```

Comment:
What's this for?

Reply:
I mean xlrd

Reply:
You can remove it. I had that required when running it in the notebook from before so I thought it was still needed. Thanks for doing refactors. I learned a lot